crates/examples/examples/common/lib.rs

```diff
@@ -12,8 +12,8 @@ pub fn small_network_config(
             r.with_chain("rococo-local")
                 .with_default_command("polkadot")
                 .with_default_image("docker.io/parity/polkadot:latest")
-                .with_node(|node| node.with_name("alice"))
-                .with_node(|node| node.with_name("bob"))
+                .with_validator(|node| node.with_name("alice"))
+                .with_fullnode(|node| node.with_name("bob"))
```

Comment:
why we change this (and the others) to be fullnodes? I think we should keep as validators and if you want include a new _fullnode_ extra